docs/Growing.md

```diff
@@ -117,6 +117,132 @@ functionality and then moves on to producing code that passes these tests. There
 having good test coverage and code being designed for testability, which will in turn often prevent future changes from
 breaking old code. Much of mojo.js was developed using TDD.
 
+## Prototype
+
+An important difference between mojo.js and other web frameworks is that it can operate in two modes, both as a 
+full-fledged web framework in single-file mode, and as a micro web framework optimized for rapid prototyping.
```

Comment:
I find the sentence `...both as a  full-fledged web framework in single-file mode, and as a micro web framework optimized for rapid prototyping.` rather confusing. What even is a ` full-fledged web framework in single-file mode`?

Reply:
I agree, I got the order mixed up there when rewriting the sentence to make sense in mojo.js context. Will push an update.